sandbox/pages/HomePage.vue

```diff
@@ -1,43 +1,157 @@
 <template>
-  <h1>Sandbox Components</h1>
-  <div class="app-layout">
-    <div class="nav-container">
-      <ul>
-        <li>
-          <router-link :to="{ name: 'home'}">
-            Home
-          </router-link>
-        </li>
-        <li>
-          <router-link :to="{ name: 'alert'}">
-            KAlert
-          </router-link>
-        </li>
-        <li>
-          <router-link :to="{ name: 'button'}">
-            KButton
-          </router-link>
-        </li>
-      </ul>
+  <div class="sandbox-header">
+    <router-link
+      class="sandbox-link"
+      :to="{ name: 'home'}"
+    >
+      <h1>
+        <span class="hidden-on-mobile">Kongponents </span>Sandbox
```

Comment:
suggestion: let's keep the word "Kongponents" showing even on mobile -- this will matter more later :)

Reply:
Done